### ISSUE TITLE
Downloaded data from CKB chart need to be clarified for the data meaning 

### ISSUE THREAD

**Issue by EmmaZHONG1024** [nervosnet/ckb-explorer-public-issues#2]
See data downloaded from CKB chart, there are three columns without titles. The first column is meaningless with 10 digits that starts with 16457. The second and third columns are  ambiguous for their meanings, which is hard to distinguish daily moving average from weekly moving average. Clarification need to be done with these data. Titles, date-time, daily and weekly moving average time are suggested to fulfill this downloaded data file. Thanks in advance.
<img width="479" alt="截屏2022-03-31 上午11 50 32" src="https://user-images.githubusercontent.com/97072854/160972822-4814dde9-d212-4aed-86a2-4ad2b80af682.png">

**Comment by Keith-CY** [comment]
> And in EN mode, the chart shows Chinese instead of English.
I visited the chart in EN mode and found it all in English
![image](https://user-images.githubusercontent.com/7271329/161184541-0be48672-6f88-4d85-bab4-36c3f5697264.png)

While the downloaded data has no text
[hash-rate.csv](https://github.com/nervosnet/ckb-explorer-public-issues/files/8394265/hash-rate.csv)

Where are the Chinese texts you mentioned above?

**Comment by Keith-CY** [comment]
> Suggest adding titles and datetime

Indeed, the timestamp is in incorrect format while the whole table needs a table header, I'll update it later.

**Comment by Keith-CY** [comment]
I've checked the internal issues and found it's actually a feature to remove the table header

> >导出的文件格式就是 csv 吗？还是说有其他的需求场景？
> 
> csv, 纯数据，无表头

Ref: https://github.com/nervosnet/ckb-explorer-internal/issues/603#issuecomment-635127959
